packages/storybook/stories/va-breadcrumbs-uswds.stories.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+import { VaBreadcrumbs } from '@department-of-veterans-affairs/web-components/react-bindings';
+
+const breadcrumbsDocs = getWebComponentDocs('va-breadcrumbs');
```

Comment:
Can you add this mapping line to the Storybook file?

`VaBreadcrumbs.displayName = 'VaBreadcrumbs';`

This will display the React binding name in the documentation instead of it saying `forwardRef`.

![Screenshot 2023-07-28 at 11 28 09 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/9fcdda52-8f70-44c6-a289-fbc0131076a5)